model/attributes.yaml

```diff
@@ -1,187 +1,208 @@
+        examples: [ "frontend-deployment", "s3-bucket-secrets", "admin-user" ]
+        requirement_level: opt_in
+      - id: policy.target.type
         type: string
         stability: development
         brief: >
-          The outcome of the policy enforcement.
-        examples: [ "success", "fail", "error" ]
-        note: |
-          This is required if the policy enforcement action is not "audit".
-        requirement_level: recommended
-      - id: policy.enforcement.action
+          Type of the resource or entity being evaluated or enforced against.
+        examples: [ "deployment", "resource", "user", "configuration" ]
+        requirement_level: opt_in
+      - id: policy.target.environment
         type: string
         stability: development
         brief: >
-          The action take by the policy enforcement.
-        examples: [ "block", "mutate", "audit" ]
-        requirement_level: required
+          Environment where the target resource or entity exists.
+        examples: [ "production", "staging", "development" ]
+        requirement_level: opt_in
+
   - id: registry.compliance
     type: attribute_group
-    display_name: Compliance Evidence Attributes
+    display_name: Compliance Assessment Attributes
     brief: >
-      This group describes attributes specific compliance audit logs. The attributes are inspired by
-      the OCSF Compliance Finding object.
+      Attributes added by compliance assessment tools to map policy results to compliance frameworks.
+      Provides compliance context, risk assessment, and regulatory mapping for audit and reporting.
+      Maps to GEMARA Layer 5 (Enforcement) for Policy-as-Code workflows.
     attributes:
-      - id: compliance.status
+      - id: compliance.control.id
         type: string
         stability: development
         brief: >
-          The normalized status identifier of the compliance check.
+          Unique identifier for the security control and assessment requirement
+          being assessed.
         examples:
-          [
-            "Unknown",
-            "Pass",
-            "Warning",
-            "Fail",
-            "Other",
-          ]
+          [ "OSPS-QA-07.01" ]
         requirement_level: required
-      - id: compliance.audit.id
+      - id: compliance.control.category
         type: string
         stability: development
         brief: >
-          A unique identifier for a specific audit or assessment.
+          Category or family that the security control belongs to.
         examples:
-          ["my_audit_id"]
-        note: |
-          This aligns with a id field of a `gemara` Evaluation or Enforcement Plan. The goal to allow batching
-          and correlation of a set of findings.
+          [ "Access Control", "Quality"]
         requirement_level: recommended
-      - id: compliance.control.id
+      - id: compliance.control.catalog.id
```

Comment:
I think we may have cases where a control is mapped to multiple "more generic" catalogs, similar to `frameworks` below. Wouldn't be good to allow a `string[]` here?